chore: add more explanations about aws secret with slash in kong gateway (#7728)
Kong/docs.konghq.com: app/_src/gateway/kong-enterprise/secrets-management/backends/aws-sm.md

{vault://aws/secret-name/foo#2}
```

{:.note}

> > **Note:** The slash symbol `/` is a valid character for the secret name in AWS SecretsManager. If you want to reference a secret name that starts with slash or have two consecutive slashes, make sure those slashes inside the name are transformed into URL-encoded format. For example a secret named `/secret/key` should be referenced as `{vault://aws/%2F/secret/key}`, and a secret named `secret/path//aaa/key` should be referenced as `{vault://aws/secret/path/%2F/aaa/key}`. Since {{site.base_gateway}} tries to resolve the secret reference as a valid URL, using slash instead of URL-encoded slash will result in unexpected secret name fetching.

Reviewer: ```suggestion
> **Note:** The slash symbol (`/`) is a valid character for the secret name in AWS SecretsManager. If you want to reference a secret name that starts with a slash or has two consecutive slashes, transform the slashes in the name into URL-encoded format. For example:
 *  A secret named `/secret/key` should be referenced as `{vault://aws/%2F/secret/key}`
 *  A secret named `secret/path//aaa/key` should be referenced as `{vault://aws/secret/path/%2F/aaa/key}`
 Since {{site.base_gateway}} tries to resolve the secret reference as a valid URL, using a slash instead of a URL-encoded slash will result in unexpected secret name fetching.
```

{vault://aws/secret-name/foo#2}
```

{:.note}
> **Note:** The slash symbol (`/`) is a valid character for the secret name in AWS SecretsManager. If you want to reference a secret name that starts with a slash or has two consecutive slashes, transform one of the slashes in the name into URL-encoded format. For example:
 *  A secret named `/secret/key` should be referenced as `{vault://aws/%2Fsecret/key}`
 *  A secret named `secret/path//aaa/key` should be referenced as `{vault://aws/secret/path/%2Faaa/key}`

>  Since {{site.base_gateway}} tries to resolve the secret reference as a valid URL, using a slash instead of a URL-encoded slash will result in unexpected secret name fetching.

Reviewer: ```suggestion
>
> Since {{site.base_gateway}} tries to resolve the secret reference as a valid URL, using a slash instead of a URL-encoded slash will result in unexpected secret name fetching.
```
Fixing a whitespace issue that I didn't see before.
